lib/bootloader_setup.pm

```diff
@@ -35,17 +36,33 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+# 'up' also works in textmode and UEFI menues.
+sub stop_grub_timeout {
+    send_key 'up';
```

Comment:
In general: Yes, of course. e.g. see http://lord.arch/tests/6235#step/grub_test/1 which is pressing 'up' to stop the timeout before confirming the entry. I am confused why you ask. There was a problem that @sysrich mentioned within the inst-bootmenu when initially pressing 'up' that you would not end up on the expected entry for booting from local disk but that does not apply here and also I did not observe any problem in all the ppc64le jobs I have tested. But feel free to see my description if I missed a scenario which is expected to behave very different.